Content.Server/GameTicking/Rules/Components/RevolutionaryRuleComponent.cs

```diff
@@ -0,0 +1,82 @@
+using Content.Shared.Roles;
+using Robust.Shared.Audio;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.GameTicking.Rules.Components;
+
+/// <summary>
+/// Stores mostly info for starting the gamerule and for passing info at the end of the round.
+/// </summary>
+
+[RegisterComponent, Access(typeof(RevolutionaryRuleSystem))]
```

Comment:
Don't put blank lines between attributes and doc comments, this also doesn't explain the rule.

Reply:
Unless I read it wrong Emo told me to make a new line in between those as it is easier to read in one of the comments they made.